bedrock/firefox/templates/firefox/os/index.html

```diff
@@ -207,9 +213,21 @@
         <p class="footnote">{{_('* Pre-installed apps may vary by manufacturers and operators.')}}</p>
       </div>
     </section>
+    {% endblock %}
+
+    {% block fxox_hub %}
+    {% endblock %}
+
+    {% block fxox_mdn %}
+    {% endblock %}
+
+    {% block fxox_try %}
+    {% endblock %}
 
     {% set feed_links = firefox_os_feed_links(LANG) %}
```

Comment:
I think this line and the conditional block below (`{% if feed_links %}`) need to move inside the `{% block fxox_news %}` block. Unless I do this, the other pages seem to throw the error mentioned earlier.